script/install.sh

```diff
@@ -458,11 +463,31 @@ resolve_latest() {
             logV "Determining latest ldc-beta version ($url)."
             COMPILER="ldc-$(fetch $url)"
             ;;
+        ldc-*)
+            ;;
         gdc)
             local url=http://gdcproject.org/downloads/LATEST
             logV "Determing latest gdc version ($url)."
             COMPILER="gdc-$(fetch $url)"
             ;;
+        gdc-*)
+            ;;
+        dub)
+            local mirrors=(
+                "http://code.dlang.org/download/LATEST"
+                "http://dlang.github.io/dub/LATEST"
+            )
+            logV "Determining latest dub version (${mirrors[0]})."
+            DUB_VERSION="$(fetch "${mirrors[@]}")"
+            local DUB="dub-${DUB_VERSION}"
+            if [ -z "$1" ] ; then
+                COMPILER="$DUB"
```

Comment:
I wasn't sure on the best way to deal with this.
The problem is that I wanted to reuse the logic here in `install_dub`, but in `install dmd` path, we can't set `COMPILER`, but for `install dub` we should.

The missing coverage is because currently we only use specific versions on for the CI.